tasks/src/main/scala/dagr/tasks/JarTask.scala

```diff
@@ -27,8 +27,22 @@ import java.nio.file.Path
 
 import dagr.core.execsystem.Memory
 
+import scala.annotation.tailrec
 import scala.collection.mutable.ListBuffer
 
+object JarTask {
+  /** Looks up the first super class that is does not have "\$anon\$" in its name. */
+  @tailrec
+  final def findCommandName(clazzOption: Option[Class[_]], errClassName: Option[String] = None): String = {
```

Comment:
It feels a bit weird that the method takes an `Option[Class]` as input - because an outside caller should never invoke it with `None`.